### ISSUE TITLE
EnergyPlus is not creating the eplusout.err file

### ISSUE THREAD

**Issue by rwadhwa910** [NREL/OpenStudio#4274]
Issue overview

I am running an Openstudio model and the simulation fails with the error EnergyPlus failed and did not create an eplusout.end file. Check the stdout-energyplus log. The Energyplus log file has nothing in it. and the eplusout.err file is totally blank which makes it quite difficult for the user to find what the issue is, although when I ran the in.idf file using energyplus I was able to find out the issue and then I was able to resolve but it will be easy if the OS simulation shows that error.

## Steps to Reproduce
I am attaching the link to the model, which if you run will find the blank eplusout.err file.

https://drive.google.com/file/d/1gSqE02u3bksz6AdPZIy6mXlUAKx0PpX2/view?usp=sharing

## Possible Solution
The error should be visible after the OS simulation fail rather than forcing the user to run the energyplus simulation.

### Environment

OS 3.1.0

**Comment by jmarrec** [comment]
I cannot reproduce.

I extracted your directory. And I am re-rerunning the simulation.

```bash 
(py39)julien@4274_eplusout (master *%>)$ ls
 Building_Model_v5/  Building_Model_v5.osm
(py39)julien@4274_eplusout (master *%>)$ cd Building_Model_v5/

# Delete the run/ directory
(py39)julien@Building_Model_v5 (master *%>)$ /bin/rm -Rf run/
(py39)julien@Building_Model_v5 (master *%>)$ openstudio --version
3.1.0+82d3ea2978

# Re run model
(py39)julien@Building_Model_v5 (master *%>)$ openstudio run -w workflow.osw 
[15:05:49.021486 WARN] Removing 1 collinear vertices from Face 0.
[15:05:49.022558 WARN] Removing 2 collinear vertices from Face 30.
[...]
0.001554422 Duration for last step, 860
[15:05:53.020009 WARN] EnergyPlus returned a non-zero exit code. Check the stdout-energyplus log.
[15:05:53.020236 ERROR] :/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/util/energyplus.rb failed with EnergyPlus Terminated with a Fatal Error. Check eplusout.err log., :/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/util/energyplus.rb:187:in `call_energyplus'
:/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/jobs/run_energyplus.rb:63:in `perform'
:/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/run.rb:292:in `step'
:/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/run.rb:234:in `run'
:/openstudio_cli.rb:971:in `execute'
:/openstudio_cli.rb:761:in `execute'
:/openstudio_cli.rb:1755:in `<main>'
eval:175:in `eval'
eval:175:in `require_embedded_absolute'
eval:160:in `block in require_embedded'
eval:154:in `each'
eval:154:in `require_embedded'
eval:113:in `require'
eval:3:in `<main>'
[15:05:53.021118 ERROR] Found error in state 'simulation' with message [":/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/util/energyplus.rb failed with EnergyPlus Terminated with a Fatal Error. Check eplusout.err log., :/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/util/energyplus.rb:187:in `call_energyplus'\n:/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/jobs/run_energyplus.rb:63:in `perform'\n:/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/run.rb:292:in `step'\n:/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/run.rb:234:in `run'\n:/openstudio_cli.rb:971:in `execute'\n:/openstudio_cli.rb:761:in `execute'\n:/openstudio_cli.rb:1755:in `<main>'\neval:175:in `eval'\neval:175:in `require_embedded_absolute'\neval:160:in `block in require_embedded'\neval:154:in `each'\neval:154:in `require_embedded'\neval:113:in `require'\neval:3:in `<main>'::/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/util/energyplus.rb:205:in `rescue in call_energyplus'\n:/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/util/energyplus.rb:128:in `call_energyplus'\n:/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/jobs/run_energyplus.rb:63:in `perform'\n:/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/run.rb:292:in `step'\n:/ruby/2.5.0/gems/openstudio-workflow-2.1.0/lib/openstudio/workflow/run.rb:234:in `run'\n:/openstudio_cli.rb:971:in `execute'\n:/openstudio_cli.rb:761:in `execute'\n:/openstudio_cli.rb:1755:in `<main>'\neval:175:in `eval'\neval:175:in `require_embedded_absolute'\neval:160:in `block in require_embedded'\neval:154:in `each'\neval:154:in `require_embedded'\neval:113:in `require'\neval:3:in `<main>'"]}
```

```bash
(py39)julien@Building_Model_v5 (master *%>)$ cat run/eplusout.err 
Program Version,EnergyPlus, Version 9.4.0-998c4b761e, YMD=2021.04.07 15:05,
   ************* Beginning Zone Sizing Calculations
   ** Warning ** Weather file location will be used rather than entered (IDF) Location object.
   **   ~~~   ** ..Location object=COLLEGE STATION EASTERWOOD FL
   **   ~~~   ** ..Weather File Location=College Station Easterwood Fl TX USA TMY3 WMO#=722445
   **   ~~~   ** ..due to location differences, Latitude difference=[9.19] degrees, Longitude difference=[136.14] degrees.
   **   ~~~   ** ..Time Zone difference=[0.0] hour(s), Elevation difference=[0.00] percent, [0.00] meters.
   ** Warning ** ProcessScheduleInput: Schedule:Constant="ALWAYS OFF DISCRETE", Blank Schedule Type Limits Name input -- will not be validated.
   ** Warning ** ProcessScheduleInput: Schedule:Constant="ALWAYS ON CONTINUOUS", Blank Schedule Type Limits Name input -- will not be validated.
   ** Warning ** GetHTSurfaceData: Surfaces with interface to Ground found but no "Ground Temperatures" were input.
   **   ~~~   ** Found first in surface=FACE 191
   **   ~~~   ** Defaults, constant throughout the year of (18.0) will be used.
   ** Severe  ** GetSurfaceData: Zone has no surfaces, Zone=WW_N_INTERIOR
   **  Fatal  ** GetSurfaceData: Errors discovered, program terminates.
   ...Summary of Errors that led to program termination:
   ..... Reference severe error count=1
   ..... Last severe error=GetSurfaceData: Zone has no surfaces, Zone=WW_N_INTERIOR
   ************* Warning:  Node connection errors not checked - most system input has not been read (see previous warning).
   ************* Fatal error -- final processing.  Program exited before simulations began.  See previous error messages.
   ************* EnergyPlus Warmup Error Summary. During Warmup: 0 Warning; 0 Severe Errors.
   ************* EnergyPlus Sizing Error Summary. During Sizing: 4 Warning; 1 Severe Errors.
   ************* EnergyPlus Terminated--Fatal Error Detected. 4 Warning; 1 Severe Errors; Elapsed Time=00hr 00min  0.34sec
```


@rwadhwa910 could you try the same steps please?

**Comment by rwadhwa910** [comment]
@jmarrec I ran the model using the OS GUI again and I could not see anything in eplusout.err file.If you run the .osm model that I have attached you won't be able to find anything in the eplusout.err file. Although when I run the in.idf file I am able to get the error as mentioned above. I have attached the eplusout.err out.osw and workflow.osw image with this comment.

![1](https://user-images.githubusercontent.com/63880759/113970688-ec7b7a00-97fc-11eb-869b-953295ca282b.PNG)

![2](https://user-images.githubusercontent.com/63880759/113970693-edaca700-97fc-11eb-9b1e-ae63d3031942.PNG)

![Capture](https://user-images.githubusercontent.com/63880759/113970695-ee453d80-97fc-11eb-97d9-1c969ee3498d.PNG)

**Comment by jmarrec** [comment]
I used the OpenStudioApp 1.1.0 just to confirm, I can still see the eplusout.err with content. That's on Ubuntu.

So I fired up a windows VM, and I can reproduce the blank file running via OpenStudio 3.1.0.

But if I run the in.idf via EnergyPlus V9-4-0 official, I get the same issue.

```bash
C:\Users\julien\Downloads\Building_Model_v5 (1)\Building_Model_v5\run>C:\EnergyPlusV9-4-0\energyplus.exe
EnergyPlus Starting
EnergyPlus, Version 9.4.0-998c4b761e, YMD=2021.04.08 09:49

C:\Users\julien\Downloads\Building_Model_v5 (1)\Building_Model_v5\run>
```

With EP Launch it's clearer:


![image](https://user-images.githubusercontent.com/5479063/113992216-83732100-9853-11eb-9286-e08e92765b07.png)


So I think it's an EnergyPlus issue, not an OpenStudio one.
